packages/stockflux-components/src/components/buttons/borderless-buttons/app-shortcuts/External.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Launchers } from 'stockflux-core';
+import RoundButton from '../../round-button/RoundButton';
+
+export default ({ manifest, small }) => {
+  if (small) {
+    return (
+      <RoundButton
+        className="shortcut external"
+        onClick={() => Launchers.launchChildWindow(manifest)}
+        small={small}
+      >
+        {manifest.description}
+      </RoundButton>
+    );
```

Comment:
I don't think we need the small button here as external apps will only appear in the launcher and not the search results window.